core/templates/dev/head/pages/exploration_editor/editor_tab/StateInteraction.js

```diff
@@ -154,6 +154,9 @@ oppia.controller('StateInteraction', [
           updateDefaultTerminalStateContentIfEmpty();
         }
         stateInteractionIdService.saveDisplayedValue();
+        if (stateSolutionService.displayed) {
```

Comment:
I'm slightly confused about why this is here. Is the solution editor in the customization modal?

In any case, what if the user deletes an existing solution? This should trigger, presumably, but an empty solution won't pass the "if" check?

If the user changes the interaction, what happens to the existing solution?

Are there e2e tests that check these things? E2e and karma tests should be part of this PR.